index.html

```diff
@@ -23,16 +23,16 @@
     <th>Non-Member</th>
   </tr>
   <tr>
-    <td>*Instant News</td>
-    <td>-Just News</td>
+    <td class="mm">*Instant News</td>

```

Comment:
Try and be a little more specific with the class / id names for your CSS styling.  Additionally, if every first td within a tr is being changed, you should look into the nth child selector. https://www.w3schools.com/cssref/sel_nth-child.asp